uptane-standard.md

```diff
@@ -738,24 +716,13 @@ An ECU version report is a metadata structure that MUST contain the following in
   * The signature of the hash
 * A payload containing:
   * The ECU's unique identifier (e.g. the serial number)
-  * The latest time downloaded from the time server, if the time server is implemented
```

Comment:
Is some notion of time needed / useful here?  It's a matter of saying when this data was generated.  I suppose you are saying that the nonce covers this, but perhaps being clearer about the choices.